rapids/README.md

```diff
@@ -0,0 +1,96 @@
+
+Our initialization actions do the following:
+1. [install nvidia GPU driver](install-gpu-driver.sh)
+2. [install RAPIDS](install-rapids.sh) - [installs miniconda](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/conda), and [conda packages](conda-environment.yml)
+3. [start dask-scheduler and dask-cuda-workers](launch-dask.sh)
+
+## Using this initialization action
+You can use this initialization action to create a new Dataproc cluster with RAPIDS installed:
+
+1. Using the `gcloud` command to create a new cluster with this initialization action. The following command will create a new cluster named `<CLUSTER_NAME>`.
+
+    ```bash
+    DATAPROC_BUCKET=dataproc-initialization-actions
+
+    gcloud beta dataproc clusters create <CLUSTER_NAME> \
+    --zone us-east1-c \
+    --master-accelerator type=nvidia-tesla-t4,count=4 \
+    --master-machine-type n1-standard-32 \
+    --worker-accelerator type=nvidia-tesla-t4,count=4 \
+    --worker-machine-type n1-standard-32 \
+    --initialization-actions gs://$DATAPROC_BUCKET/rapids/install-gpu-driver.sh,gs://$DATAPROC_BUCKET/rapids/install-rapids.sh,gs://$DATAPROC_BUCKET/rapids/launch-dask.sh
+    ```
+
+1. Once the cluster has been created, the Dask scheduler listens for workers on port `8786`, and its status dashboard is on port `8787` on the Dataproc master node. These ports can be changed by modifying the [launch-dask.sh](launch-dask.sh) script.
+
+To connect to the Dask web interface, you will need to create an SSH tunnel as described in the [dataproc web interfaces](https://cloud.google.com/dataproc/cluster-web-interfaces) documentation. You can also connect using the [Dask Client Python API](http://distributed.dask.org/en/latest/client.html) from a notebook, or a from a plain Python script or interpreter session.
+
+An [example notebook](notebooks/NYCTaxi-E2E.ipynb) is provided that demonstrates end to end data pre-processing (cuDF & Dask) and model training (XGBoost) with RAPIDS APIs. Additional example notebooks [are available](https://github.com/rapidsai/notebooks). See the [RAPIDS documentation](https://docs.rapids.ai/) for API details.
+
+RAPIDS is a relatively young project with APIs evolving quickly. If you encounter unexpected errors or have feature requests, please file them at the relevant [RAPIDS repo](https://github.com/rapidsai).
+
+### Options
+
+By default, the master node also runs dask-cuda-workers. This is useful for smaller scale jobs- processes run on 4 GPUs in a single node will usually be more performant than when run on the same number of GPUs in separate server nodes (due to higher communication costs).
+
+If you want to save the master's GPU(s) for other purposes, this behavior is configurable via a metadata key using `--metadata`.
+
+* `master-worker=false` - whether to run dask-cuda-workers on the master node
```

Comment:
can you use more descriptive name here? Something like: `run-cuda-worker-on-master`?